<?xml version="1.0"?>
<launch>

    <node pkg="roomba_dynamixel_controller" type="teleop_dynamixel_angle" name="teleop_dynamixel_angle" output="screen" />
    <node pkg="roomba_dynamixel_controller" type="roomba_dynamixel_controller" name="roomab_dynamixel_controller" >
        <param name="execution_time" value="0.5" type="double" />
        <param name="offset_angle" value="0.00" type="double" />
    </node>

</launch>
